srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -807,11 +799,23 @@ struct {
   uint EKT_Key_Value<1..256>;
   uint EKT_Master_Salt<1..256>;
   uint16 EKT_SPI;
+  uint24 EKT_TTL; 
 } EKTkey;
 ]]></artwork>
 
 			</figure>
 
+            <t> 
+             <list style="hanging">
+            <t hangText="EKT_TTL:">The maximum amount of time, in
+            seconds, that this EKTkey can be used. The EKTkey in this
+            message MUST NOT be used for encrypting or decrypting
```

Comment:
some rewording on the lines : "The EKT Cipher associated with a given EKTKey MUST NOT be used for encrypting and decrypting information after the TTL expires." -- I am trying not to use EKTKey word above since it is just a composite not the key itself